src/mono/mono/component/debugger-agent.c

```diff
@@ -414,7 +414,7 @@ static gboolean buffer_replies;
 	DebuggerTlsData *tls; \
 	tls = (DebuggerTlsData *)mono_native_tls_get_value (debugger_tls_id);
 #else
-#define GET_TLS_DATA_FROM_THREAD(thread) \
+#define GET_TLS_DATA_FROM_THREAD(...) \
```

Comment:
Might be better to disable some of these new llvm warnings, the warning workarounds make the code harder to read.

Reply:
I have added comment to this line to improve the readability. Hopefully that improves it.